pex/venv_bin_path.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import
+
+
+class BinPath(object):
```

Comment:
This looks similar to `Enum` - once we drop Py2, would we want to switch this to that? If so, a TODO would be great.

Reply:
Yes, this should go to enum as should InheritPath. This just moved out of pex.tools.commands.venv so I'll leave as-is. There are alot of things that can change once we drop Python 2.7 including namedtuple -> dataclass and more. That will be a whole project.